fuzzers/centipede/fuzzer.py

```diff
@@ -21,15 +21,25 @@
 
 def build():
     """Build benchmark."""
-    # TODO(Dongge): Build targets with sanitizers.
-    cflags = [
-        '-fno-builtin',
-        '-gline-tables-only',
+    san_cflags = ['-fsanitize-coverage=trace-loads']
+
+    link_cflags = [
         '-ldl',
         '-lrt',
         '-lpthread',
-        '-fsanitize-coverage=trace-loads,trace-pc-guard,trace-cmp,pc-table',
+        '/lib/weak.o',
     ]
+
+    # TODO(Dongge): Build targets with sanitizers.
+    try:
+        centipede_cflags = subprocess.run(
+            ['cat', '/src/centipede/clang-flags.txt'],
+            stdout=subprocess.PIPE,
+            check=True).stdout.decode('utf-8').split('\n')
+    except subprocess.CalledProcessError as err:
+        print(err)
```

Comment:
Also this is not a good way to handle the error. Do we want to just print the error and continue? No, we should fail.